Source/Main.py

```diff
@@ -6,12 +6,12 @@
 import os
 import GUI as gui
 
-coolGuy = gen.Genetic(mutRate=.008, mutType="Random", popSize=10, crossType="Crossy", cutoff=5, gridSize=7)
+coolGuy = gen.Genetic(mutRate=.008, mutType="Random", popSize=10, crossType="Crossy", cutoff=5, gridSize=15)
 
 #Get directory that the test files are in
-TestPath = "../TestData/"
```

Comment:
Do you not need the `../`? When I run without it crashes. Maybe because you are running using the play button instead of from the terminal?

Reply:
Yeah I just TestData/7x7.txt and it works